solr/core/src/java/org/apache/solr/handler/admin/StatusOp.java

```diff
@@ -39,6 +42,14 @@ public void execute(CoreAdminHandler.CallInfo it) throws Exception {
     String reconcile = params.get(CoreAdminParams.RECONCILE_THRESHOLD);
     boolean isIndexInfoNeeded = Boolean.parseBoolean(null == indexInfo ? "true" : indexInfo);
     NamedList<Object> status = new SimpleOrderedMap<>();
+    NamedList<Object> nodeStatus = new SimpleOrderedMap<>();
+    if (it.handler.coreContainer != null) {
+      Path solrDataHome = it.handler.coreContainer.getCoreRootDirectory();
```

Comment:
`coreContainer.getNodeConfig().getSolrDataHome()` might seem like the more correct choice, but that can be null (in dev envs, and I'm guessing it's null if you don't explicitly set it). The `coreRootDirectory` is where all the solr cores are created, so this should be the same thing for our purposes. Will definitely test this.

Reply:
`coreRootDirectory` works on local and playpen!